docs/troubleshooting.md

```diff
@@ -107,3 +107,6 @@ more API calls (which is one reason a manifest config is preferred).
 3. Iterate through the latest tags and compare against commit
    SHAs seen in the last 250 commits on the branch. This is to
    ensure that the tag happened on this target branch.
+
+### What if my release PR is merged with `autorelease:closed`?
```

Comment:
This heading is the cause of a problem "Why my merged release pull request does not trigger the release job?"